src/less/components.less

```diff
@@ -45,16 +45,16 @@
 @import 'components/pills';
 @import 'components/popovers';
 @import 'components/radios';
-@import 'components/reveals';
-@import 'components/search';
-@import 'components/selectors';
+@import 'components/reveal/index';
+@import 'components/search/index';
+@import 'components/selector/index';
```

Comment:
sub-directory should be `src/less/components/selector-strip` to match `docs/components/selector-strip`

Reply:
Renamed `src/less/components/selector/*` -> `src/less/components/selector-strip/*`.